mycroft/client/speech/listener.py

```diff
@@ -57,110 +56,6 @@ def run(self):
                     self.emitter.emit("recognizer_loop:ioerror", ex)
 
 
-class WakewordExtractor:
```

Comment:
Just to confirm (because it's difficult to tell from the diff alone), this code was just moved, not moved and modified?